src/main/java/solvd/laba/ermakovich/hf/kafka/KafkaConsumerConfig.java

```diff
@@ -48,6 +48,12 @@ protected Map<String, Object> kafkaConsumerProperties() {
         kafkaPropertiesMap.put(ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG,
                 new XmlXPath(configPath, "valueDeserializer", resourceLoader)
                         .getText());
+        kafkaPropertiesMap.put(JsonDeserializer.USE_TYPE_INFO_HEADERS,
+                new XmlXPath(configPath, "useTypeInfoHeaders", resourceLoader)
```

Comment:
to don't create your own Xpath, you can use it from `eo-kafka` version `0.0.2`
[TextXpath](https://github.com/eo-cqrs/eo-kafka/blob/master/src/main/java/io/github/eocqrs/kafka/xml/TextXpath.java)